apps/editors/views.py

```diff
@@ -558,8 +558,8 @@ def review(request, addon):
     form = forms.get_review_form(request.POST or None, request=request,
                                  addon=addon, version=version)
 
-    queue_type = (form.helper.review_type if form.helper.review_type
-                  != 'preliminary' else 'prelim')
+    queue_type = ('prelim' if form.helper.review_type == 'preliminary'
+                  form.helper.review_type)
```

Comment:
You must have misread, this never happened.

This is not the stupid mistake you're looking for https://bloodstainedink.files.wordpress.com/2012/06/star-wars-cap-33.jpg